spec/Classes.tex

```diff
@@ -106,7 +108,9 @@ \subsection{Class Types}
 \end{syntax}
 
```

Comment:
Fully specified is the term the spec used in the "Class Types" section. I'm just trying to fit in.
See a whole section 15.1.1 and let me know which term you prefer after reviewing it. I will change to either.
It just says "fully specified" 2 lines up.

I'm happy to adjust the wording as you describe - we just need to choose "fully specified" or "fully instantiated".